### PR TITLE
REST API snake_case to camelCase

### PR DESCRIPTION
**Pull request by yorma**
change snake_case to camelCase for https://github.com/Comcast/traffic_control/issues/1509

### CODE REVIEW

**Comment by yorma** [comment]
@mitchell852 @dangogh, could you review this PR and https://github.com/Comcast/traffic_control/pull/1544 ?

**Comment by dangogh** [comment]
in this case, you could have just added a comment here to wait to merge and added another commit to your branch rather than a separate PR.

I'll try and review both of these on Monday.   Thanks!

**Comment by yorma** [comment]
That's ok, I separated PR is to avoid have too many APIs in a PR. thanks.

**Comment by dangogh** [comment]
ahh..   I see..   thanks.

File: traffic_ops/app/t/api/1.2/physlocation.t [hunk 1]
```diff
@@ -41,10 +41,10 @@ ok $t->post_ok( '/login', => form => { u => Test::TestHelper::ADMIN_USER, p => T
...
-        "short_name" => "physloc1"})->status_is(200)->or( sub { diag $t->tx->res->content->asset->{content}; } )
+        "shortName" => "physloc1"})->status_is(200)->or( sub { diag $t->tx->res->content->asset->{content}; } )
 	->json_is( "/response/name" => "physical location1" )
-	->json_is( "/response/short_name" => "physloc1" )
-    ->json_is( "/response/region_name" => "Denver Region" )
+	->json_is( "/response/shortName" => "physloc1" )
+    ->json_is( "/response/regionName" => "Denver Region" )
             , 'Does the physical location details return?';
 ok $t->post_ok('/api/1.2/regions/non_region/phys_locations' => {Accept => 'application/json'} => json => {
         "name" => "physical location1",
```
Comment (dangogh) [inline traffic_ops/app/t/api/1.2/physlocation.t]: missed this short_name

Comment (yorma) [inline traffic_ops/app/t/api/1.2/physlocation.t]: @dangogh @mitchell852 , modified with your comments, thanks.

**Pull request merged**